Lib/pdb.py

```diff
@@ -92,8 +93,11 @@
 import itertools
 import traceback
 import linecache
+import selectors
+import threading
 import _colorize
 
+from contextlib import ExitStack
```

Comment:
I think at this point, we are importing 3 different things from `contextlib` and we used 3 lines - that's unnecessary. We could either put them in the same line, or just import `contextlib` and use atttributes.